test/unit/event/send-acknowledgement-error-event.test.js

```diff
@@ -0,0 +1,78 @@
+jest.mock('../../../app/event/send-processing-ack-error-event')
+const sendProcessingAckErrorEvent = require('../../../app/event/send-processing-ack-error-event')
+
+jest.mock('../../../app/event/send-processing-ack-invalid-bank-details-error-event')
+const sendProcessingAckInvalidBankDetailsErrorEvent = require('../../../app/event/send-processing-ack-invalid-bank-details-error-event')
+
+const mockFRN = require('../../mock-frn')
+const sendAckowledgementErrorEvent = require('../../../app/event/send-acknowledgement-error-event')
+const { BANK_ACCOUNT_ANOMALY, DAX_REJECTION } = require('../../constants/hold-categories-names')
+
+let mockAcknowledgementError
+let mockHoldCategoryName
+
+describe('send acknowledgement event', () => {
+  beforeEach(() => {
+    mockAcknowledgementError = JSON.parse(JSON.stringify(require('../../mock-acknowledgement-error')))
+    mockHoldCategoryName = JSON.parse(JSON.stringify(require('../../constants/hold-categories-names')))
+  })
+
+  afterEach(() => {
+    jest.resetAllMocks()
+    jest.resetModules()
+  })
+
+  test('should call sendProcessingAckErrorEvent when a holdCategoryName, an unsuccessful ack object and frn is given and holdCategoryName is "DAX rejection"', async () => {
+    mockHoldCategoryName = DAX_REJECTION
```

Comment:
Can use my syntax sugar trick from `processInvalid` to move this into `beforeEach` and define once